src/integrations/raiden/index.ts

```diff
@@ -167,13 +167,13 @@ export class RaidenInspector extends Inspector<RaidenAppointment> {
 
         const code: string = await this.provider.getCode(contractAddress);
         // check that the channel is a contract
-        if (code === "0x" || code === "0x00") {
+        if (!code || code === "0x" || code === "0x00") {
             throw new PublicInspectionError(`No code found at address ${contractAddress}`);
         }
```

Comment:
Is the behavior of returning "0x" or "0x00" observed, or is it a bug in ethers.js? I don't see it documented there.

Reply:
I'm only observing 0x, I'll update